mackup/applications/git.cfg

```diff
@@ -3,3 +3,7 @@ name = Git
 
 [configuration_files]
 .gitconfig
+
+[xdg_configuration_files]
+git/config
+git/credentials
```

Comment:
not sure people will like this, I've got a lot of bad feedback from trying to sync ssh keys by default.